narayana-full/src/main/assembly/bin.xml

```diff
@@ -327,8 +322,12 @@
 			<outputDirectory>quickstarts/rest-tx</outputDirectory>
 		</fileSet>
 		<fileSet>
-			<directory>../rts/lra/coordinator-quarkus/target/lib</directory>
-			<outputDirectory>rts/lra/lib</outputDirectory>
+			<directory>../rts/lra/coordinator-quarkus/target/</directory>
+			<outputDirectory>rts/lra/</outputDirectory>
+			<includes>
+				<include>lra-coordinator-runner*</include>
```

Comment:
@xstefank I am thinking that this is not just cleanup and some files will appear in the zip. Is that a correct line of thinking? If it is then we need to consider that generally and if the decision was that this file were to be included in the zip then it would be important to change the name of the zip to reflect the compatibility - like we do in BlackTie. I think this might where the assembly name is set but it's a long time ago - if we proceed in adding files to the zip please let me know if you need help to work out how to make sure the name is reflected: https://github.com/jbosstm/narayana/blob/master/blacktie/blacktie/src/main/assembly/bin.xml#L19

Reply:
sorry I must have misunderstood your previous comment. Yes, this will add lra-coordinator files, jar and optionally binary, to the zip.

Reply:
The zip would end up on our website and the script that is being edited is the one that helps product and upload that. You can find our binaries here: https://narayana.io/downloads/index.html

Reply:
If the only reason to include the Linux 64 binary of lra-coordinator is for the docker image than I would vote to build it when the docker image is created. This way we can avoid adding additional zip artifacts.